core/test.py

```diff
@@ -1,8 +1,37 @@
+from abc import ABCMeta
+
 from unittest import TestCase
 
+from core import app
+from user.models import User
+from blog.models import Article
 from .utils import clean_html
 
 
+class BaseTest(TestCase):
+    __metaclass__ = ABCMeta
+
+    def setUp(self):
+        self.client = app.test_client()
+
+    def tearDown(self):
+        User.objects.delete()
+        Article.objects.delete()
+
+    def login_user(self):
+        data = {
+            'email': 'emailtest@test.com',
+            'password': 'testtest',
+        }
+        self.client.post('/login', data=data, follow_redirects=True)
+
+    @staticmethod
```

Comment:
why making it static? you could do `self.create_user()` or even have `self.user` in he `setUp`